bitnami/cassandra/values.yaml

```diff
@@ -940,18 +941,28 @@ metrics:
 ## @param tls.resources Set container requests and limits for different resources like CPU or memory (essential for production workloads)
 ## @param tls.certificatesSecret Secret with the TLS certificates.
 ## @param tls.tlsEncryptionSecretName Secret with the encryption of the TLS certificates
+## @param tls.secretKeys.keystore Key within `tls.existingSecret` that contains the keystore file
+## @param tls.secretKeys.keystorePassword Key within `tls.passwordsSecret` that contains the keystore secret
+## @param tls.secretKeys.truststore Key within `tls.existingSecret` or `tls.existingTrustSecret` (if set) that contains the truststore file
+## @param tls.secretKeys.truststorePassword Key within `tls.passwordsSecret` that contains the truststore secret
 ##
 tls:
   internodeEncryption: none
   clientEncryption: false
   autoGenerated: false
   existingSecret: ""
+  existingTrustSecret: ""
```

Comment:
For consistency, would it be possible to rename this as `existingTruststoreSecret`?

Reply:
Yes, good idea. Done.